utils/cfutils.go

```diff
@@ -0,0 +1,222 @@
+package utils
```

Comment:
In think the `cfutils.go` should not output anything via `fmt.Fprintln` as it's a utility file that may be used in multiple places. If something fails, it should rather return an error object with an error message. With that, the caller can still decide whether to print the error message or to do something else instead.

Reply:
updated accordingly, return the message as error.